neo/Prompt/Commands/Wallet.py

```diff
@@ -55,7 +61,10 @@ def execute(self, arguments):
             return
 
         try:
-            self.execute_sub_command(item, arguments[1:])
```

Comment:
In my opinion, the original signature `execute(self, arguments)` should be respected. None could be given as arguments for commands like `CommandWalletClose`.

Reply:
Okay, I didn't know of that was preferred. I had that originally but I thought it looked kinda silly.

Reply:
I think it was just the way I originally implemented it.